iminuit/tests/test_frontend.py

```diff
@@ -1,35 +1,560 @@
-from __future__ import (absolute_import, division, print_function,
-                        unicode_literals)
+from __future__ import (absolute_import, division, print_function)
```

Comment:
Suggest to add one comment line:
```
# Here we don't import unicode_literals, because it's easiest to put in "\usepackage" below
# See https://stackoverflow.com/questions/7602171/unicode-error-unicodeescape-codec-cant-decode-bytes-string-with-u
```

Reply:
Ok, good point.